global_shortcuts.go

```diff
@@ -0,0 +1,135 @@
+package astilectron
+
+import (
+	"context"
+	"sync"
+)
+
+const (
+	EventNameGlobalShortcutsCmdRegister          = "global.shortcuts.cmd.register"
+	EventNameGlobalShortcutsCmdIsRegistered      = "global.shortcuts.cmd.is.register"
```

Comment:
Could you add the missing `ed` at the end of `global.shortcuts.cmd.is.register`?